waiter/config-full.edn

```diff
@@ -350,6 +350,11 @@
                                 "scale-factor" 1
                                 "scale-up-factor" 0.1}
 
+ ;; Service description parameters with numeric values, e.g. cpus, can be configured to have
+ ;; maximum value limits. When not configured, these defaults will be used:
+ :service-description-resource-limits {"cpus" 100
+                                       "mem" 1048576}
```

Comment:
Should we clarify with a comment that this is in MB? Also, maybe a smaller amount for this example would be more appropriate

Reply:
Added.